src/AbstractDifferentiation.jl

```diff
@@ -291,21 +295,24 @@ The pullback function `pb` accepts as input a `Tuple` of cotangents, one for eac
 If `f` has a single output, `pb` can also accept a single input instead of a 1-tuple.
 """
 function pullback_function(ab::AbstractBackend, f, xs...)
-    _, pbf = value_and_pullback_function(ab, f, xs...)
-    return pbf
+    _, pb = value_and_pullback_function(ab, f, xs...)
```

Comment:
What's the motivation for this change? `pbf` seemed to refer to "pullback function" I assume which seems to be the correct interpretation?

Reply:
I think the terms "pullback" and "pullback function" are both valid, see eg. the JAX docs. My rationale for this was to fight ambiguities, because the function returning pb is already called pullback_function. So the idea is that the pullback function returns a pullback, which is itself a function

Reply:
My feeling was that the change here actually made it less clear - `gradient` etc. return gradients, and similarly `pullback_function` returns a "pullback function" for which `pbf` seemed to be an appropriate abbreviation.

Reply:
Then we should also turn `pf` into `pff` on the pushforward side (another reason for my change). I don't have strong opinions either way, it just feels weird to me that `pullback_function` returns a `pbf` which is itself short for... pullback function

Reply:
fixed in my latest commit